test/test-debuglet.ts

```diff
@@ -377,6 +377,32 @@ describe('Debuglet', () => {
       assert.deepEqual(mergedConfig, compareConfig);
     });
 
+    it('should not start when workingDirectory is the a directory', (done) => {
+      const debug = new Debug({}, packageInfo);
+      const config = extend({}, defaultConfig, {workingDirectory: path.sep});
```

Comment:
Does passing `path.sep` accurately represent Windows root directories?

Reply:
Yes, I've check that it works on Windows and OS X.